ccd-definition/AuthorisationCaseType/AuthorisationCaseType.json

```diff
@@ -6,5 +6,25 @@
       "caseworker-civil-admin"
     ],
     "CRUD": "CRU"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "UserRole": "caseworker-approver",
+    "CRUD": "CRUD"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "UserRole": "caseworker-caa",
```

Comment:
Could we please combine caseworker approver, caseworker caa andcaseworker civil systemupdate roles as they share the same CRUD permissions

Reply:
Updated